tests/integration/models/test_cohort.py

```diff
@@ -7,8 +7,8 @@
 
 
 class CohortType(enum.Enum):
-    static = enum.auto()
-    dynamic = enum.auto()
+    static = 1
+    dynamic = 2
```

Comment:
Did you know that you don't have to use numbers for an enumeration? https://github.com/NCI-GDC/graphmanager/blob/a2e798b67e7c51440b221390066f0c686c1a8942/graph_manager/versioning/models.py#L23